### DISCUSSION TITLE
Adjusting Section size to iframe widget

### DISCUSSION THREAD

**Discussion by maciekkielbus** [Lissy93/dashy#1198] [Help and Support]
Hi. 
While configuring my dashboard i hit a hard spot. 
I try to set one section to be my local camera preview. 
Everything works fine , but there is a major problem. 
I cannot to adjust section size to my iframe window, its like collapsed the whole time and I cant get it open just to show the whole image of iframe source.  
On the image below you have a preview of my case.
<a href="https://ibb.co/ykKmbPP"><img src="https://i.ibb.co/Jp8WMtt/dashy-problem.jpg" alt="dashy-problem" border="0"></a>
I dont want to use scrolls, instead i want to see the whole image without section being collapsed.
Is there any way to do this?
Thanks in advance for answers.  :)

**Comment by CrazyWolf13** [comment]
Hi that is what I use, this works fine for me:

```
    widgets:
      - type: embed
        options:
          html: >
            <p align="center"><iframe
            src="{url}"
            frameborder='0' scrolling="no" style=" width: 540px; height: 615px;
            overflow: hidden;"></iframe></p>

```
this also centers the iframe in the section and removes scroll bar.
